doc/articles/uno-development/debugging-uno-ui.md

```diff
@@ -103,6 +103,19 @@ The Source Generation tooling diagnostics can be enabled as follows:
 
 **Make sure to remove the `UnoSourceGeneratorUnsecureBinLogEnabled` property once done.**
 
+If ever the need arises to view the generated source code of a *failing* CI build, you can perform the following steps:
+
+1. In your local branch, locate the one of build yaml files (located in the root Uno folder):
+     - .azure-devops-android-tests.yml
+     - .azure-devops-macos.yml
+     - .azure-devops-wasm-uitests.yml
+    
+2. At the bottom of the yaml files, you'll find *Publish...* tasks, right above these tasks, copy/paste the following code to create a task which will copy all generated source files and put them in an artifact for you to download
```

Comment:
I believe the "following code" is missing

Reply:
done